keras_nlp/models/t5/t5_transformer_layer.py

```diff
@@ -52,16 +54,26 @@ def __init__(
 
         if self.is_decoder:
             self.cross_attention = T5MultiHeadAttention(
-                is_decoder,
-                hidden_dim,
-                num_heads,
-                dropout,
+                is_decoder=is_decoder,
+                hidden_dim=hidden_dim,
+                key_value_dim=key_value_dim,
+                num_heads=num_heads,
+                dropout=dropout,
                 use_relative_attention_bias=False,
                 name="cross_attention",
             )
             self.cross_attention_layer_norm = T5LayerNorm(layer_norm_epsilon)
             self.cross_attention_dropout = keras.layers.Dropout(dropout)
 
+        if activation == "gelu":
```

Comment:
We actually had a recent contribution on this -> https://github.com/keras-team/keras-nlp/pull/1262

Would it work to put `"activation": "keras_nlp>gelu_approximate"` in our preset when we need it? That way `"activation": "gelu"` can still mean a consistent thing across keras (the non-approximate version), and we still have a way to specify the approximate version as a string.

Reply:
Sure thing! Added it and tested with FLAN-T5 weights and it works!